spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -57,118 +57,233 @@ If you have debug mode turned on, you can get even more information, in a line s
+[source,java]
 ----
-<dependency>
-  <groupId>org.springframework.cloud</groupId>
-  <artifactId>spring-cloud-starter-task</artifactId>
-</dependency>
-<dependency>
-  <groupId>com.h2database</groupId>
-  <artifactId>h2</artifactId>
-</dependency>
-<dependency>
-  <groupId>org.springframework.boot</groupId>
-  <artifactId>spring-boot-starter-jdbc</artifactId>
-</dependency>
+package io.spring.demo.timestamp;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.task.configuration.EnableTask;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+@EnableTask
+public class TimestampApplication {
+
+	@Bean
+	public CommandLineRunner commandLineRunner() {
+		return new TimestampCommandLineRunner();
+	}
+
+	public static void main(String[] args) {
+		SpringApplication.run(TimestampApplication.class, args);
+	}
+
+	public static class TimestampCommandLineRunner implements CommandLineRunner {
+
+		@Override
+		public void run(String... strings) throws Exception {
+			DateFormat dateFormat = new SimpleDateFormat("YYYY-MM-dd");
+			System.out.println(dateFormat.format(new Date()));
+		}
+	}
+}
 ----
-. Save your changes to `pom.xml`.
-. Open `src/main/java/io/spring/TasklabApplication.java` in either an IDE or a text editor.
-. On Line 13, add the `@EnableTask` annotation.
-+
-NOTE: If you do not use an IDE, you also need to un-comment lines 6 through 9.
-. Save your changes to `TasklabApplication.java`.
-. Open `src/main/resources/application.properties`.
-. Add the following properties:
-+
+
+While it may seem small, quite a bit is going on. For more about Spring
+Boot specifics, see the
+http://docs.spring.io/spring-boot/docs/current/reference/html/[Spring Boot reference documentation].
+
+Now let's open the `application.properties` file in `src/main/resources`. We need
+to configure two properties in `application.properties`: We need to set the application
+name (which is translated to the task name), and we need to set the logging for Spring
+Cloud Task to `DEBUG` so that we can see what's going on. The following example shows how
+to do both:
+
+[source]
 ----
 logging.level.org.springframework.cloud.task=DEBUG
-spring.application.name=lab2-task
+spring.application.name=timestamp
 ----
-+
-Setting the logging level for the task to `DEBUG` creates output to let us know that the task works.
-Setting the application name lets us store a meaningful name in the database, so that we can find our task in the `TASK_EXECUTION` table.
-. Save your change to `application.properties`.
 
-Now that we have made our changes, we naturally want to run it and see it work. To do so:
+[[getting-started-at-task]]
+==== The @EnableTask annotation
```

Comment:
Oh, cool, you do explain here. Maybe it'd be good to remove "While it may seem small, quite a bit is going on." from the previous section.

Reply:
Removed that section.